### DISCUSSION TITLE
Additional smoke test

### DISCUSSION THREAD

**Discussion by Odie-123** [nicolacimmino/PLC-14500#12] [Show and tell]
Hi, this board is really cool! I like how you did it with the Arduino Nano for convenience! I came here searching for 1-Bit CPUs, in Germany there was an educational TV Series in the 80's about a "WDR 1-Bit Computer", they also used the 14500 but programming was more cumbersome.

I built your kit but i think i will replace some LEDs with different colors, namely 4 of the 8 data bits to distinguish between OP code and data bits and maybe IN7 and OUT7 to show that they have a special meaning. You used low-current LEDs? I guess it depends on what the 4099 can drive. 8x 20mA would be too much i guess.

I have an additional smoke test for you which should test almost all buttons and LEDs, you can include it in your sources here and maybe on the Arduino board if you like:

[boardtest.asm.txt](https://github.com/nicolacimmino/PLC-14500/files/14865975/boardtest.asm.txt)

**Comment by veremenko-y** [comment]
Nice job. It's a really neat machine, and I've been looking at it. But I can't read german :(
http://wdr-1-bit-computer.talentraspel.de/index.html

**Reply by Odie-123** [reply]
You could use Google Translate for the website: https://wdr--1--bit--computer-talentraspel-de.translate.goog/index.html?_x_tr_sch=http&_x_tr_sl=de&_x_tr_tl=pl&_x_tr_hl=de&_x_tr_pto=wapp

Here is a rebuild of the WDR Computer: https://hackaday.io/project/170548-wdr-1-bit-computer

And here is a video of another rebuild: https://www.youtube.com/watch?app=desktop&v=PrpStnNA2GM

**Comment by nicolacimmino** [comment]
> [boardtest.asm.txt](https://github.com/nicolacimmino/PLC-14500/files/14865975/boardtest.asm.txt)

Hi,

thanks for the listing! Looks good, I will give it a try and most likely include it when I pack the next batch of kits.
